setup.py

```diff
@@ -45,7 +45,7 @@ def run_tests(self):
 
 setup(name='chanjo-report',
       # versions should comply with PEP440
-      version='4.2.0',
+      version='4.3.0',
```

Comment:
Ok I'll leave 4.2 here

Reply:
Actually maybe I'll set both to 4.3, makes more sense, since something actually changed from the previous version